packages/aws-cdk-lib/aws-certificatemanager/lib/dns-validated-certificate.ts

```diff
@@ -106,10 +105,7 @@ export class DnsValidatedCertificate extends CertificateBase implements ICertifi
       certificateTransparencyLoggingPreference = props.transparencyLoggingEnabled ? 'ENABLED' : 'DISABLED';
     }
 
-    const requestorFunction = new lambda.Function(this, 'CertificateRequestorFunction', {
-      code: lambda.Code.fromAsset(path.resolve(__dirname, '..', '..', 'custom-resource-handlers', 'dist', 'aws-certificatemanager', 'dns-validated-certificate-handler')),
-      handler: 'index.certificateRequestHandler',
-      runtime: lambda.Runtime.NODEJS_18_X,
+    const requestorFunction = new CertificateRequestCertificateRequestProvider(this, 'CertificateRequestorFunction', {
       timeout: cdk.Duration.minutes(15),
```

Comment:
Let's have the `timeout` default to 15 minutes.

I don't think there's a good reason to set it shorter: we trust the code, and we'd rather the code finish (for whatever reason it's taking a long time), than accidentally timeout after 3 seconds or 1 minute or what have you, and us failing the deployment.